### ISSUE TITLE
Video walk-through

### ISSUE THREAD

**Issue by sm-md-lg** [ryssbowh/craft-themes#14]
It would be really helpful if you had a video walk-through of using this plugin, from installing to configuring and seeing a page making use of a theme.

I'm having a hard time getting an overall grasp of how the plugin works.  There is lots of documentation of the parts but not so much for an overall getting a page to use a theme.

The ui is quite complicated so seeing someone use it would really help to get an overall understanding of how all the parts work together.

**Comment by piotrpog** [comment]
I agree that current documentation makes it hard to grasp initial concepts of plugins. It would be great if craft quest made video about it or maybe there was some kind of "quick start" section of docs, possibly with some helpful image graphs that show how various concepts like blocks, displayers, layouts etc. relate to each other.

**Comment by ryssbowh** [comment]
Yes I agree the github wiki makes it hard to organise such an amount of documentation.
We're working on moving all the documentation to a dedicated website, it's work in progress and will take a few months to go live. Hopefully it will make things clearer, we can make a video there then.
